docs/support.mdx

```diff
@@ -1,111 +1,35 @@
 ---
-title: Getting help
+title: Support
 description: How to find answers to questions or problems you encounter while using IBM Quantum (Platform, or on IBM Cloud) or Qiskit Runtime
 
 ---
 
-# Getting help
+# Support
 
-## IBM Quantum Support
+## How to get help
 
-Members of the IBM Quantum&trade; Network can reach out to IBM Quantum Support if they have technical difficulties, questions, or concerns. Ask your administrator or IBM&reg; representative for IBM Quantum Support contact information.
+| | |
+| --- | --- |
+| IBM Quantum&trade; Network members  | Reach out to IBM Quantum Support if you have technical difficulties, questions, or concerns. Ask your administrator or IBM&reg; representative for IBM Quantum Support contact information. |
```

Comment:
I like the idea of this chart but I think we should actually split it into 2 separate tables. One table should be focused on user type (i.e. network members, paygo users, open plan users) and another table for where to find help according to specific topics